e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js

```diff
@@ -81,4 +95,58 @@ describe('Onboarding', () => {
         // # Close the admin UI and inviting people tip
         cy.get(selectors.tipNext).click();
     });
+
+    it('MM-T4148 Takes the user through the steps of using the app using the ENTER key', () => {
```

Comment:
@stevemudie is it possible to get another id for this test case?

Reply:
I've updated MM-T4148 to read "Takes the user through the steps of using the app using the ENTER key".

Reply:
@stevemudie updated